packages/block-editor/src/components/link-control/index.js

```diff
@@ -260,11 +261,27 @@ function LinkControl( {
 				/>
 			) }
 
-			<LinkControlSettingsDrawer
-				value={ value }
-				settings={ settings }
-				onChange={ onChange }
-			/>
+			<div className="block-editor-link-control__tools">
+				<LinkControlSettingsDrawer
+					value={ value }
+					settings={ settings }
+					onChange={ onChange }
+				/>
+				{ onRemove &&
+					isFunction( onRemove ) &&
```

Comment:
I don't think we need this at all, we only support passing function anyway.

Reply:
Ok sure. I've removed this now.